config/locales/es.yml

```
@@ -143,8 +143,8 @@ es:
         title: "Reenviar instrucciones de confirmación"
         submit: "Reenviar instrucciones de confirmación"
       links:
-        sign_up: "Ingresar"
-        sign_in: "Registrarse"
+        sign_up: "Registrarse"
+        sign_in: "Ingresar"
```

@roramirez I realize this was using `"Ingresar"` before but overall it seems we use `"Iniciar Sesión"` consistently. What do you think about changing that here to the following?
```suggestion
        sign_in: "Iniciar Sesión"
```

---

I'm seems good to change this string. It's more clearly to use this way  in spanish

---

Great thanks!